frontend/packages/process-editor/src/components/Canvas/Viewer/VersionAlert/VersionAlert.tsx

```diff
@@ -0,0 +1,44 @@
+import { Alert, Heading, Link, Paragraph } from '@digdir/design-system-react';
+import classes from './VersionAlert.module.css';
+import React, { ReactNode } from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+
+export type VersionAlertProps = {
+  appLibVersion: string;
+};
+
+/**
+ * @component
+ *  Displays the alert informing the user that their version is too old
+ *
+ * @property {string}[appLibVersion] - The app-lib version the user has
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const VersionAlert = ({ appLibVersion }: VersionAlertProps): ReactNode => {
+  const { t } = useTranslation();
+
+  return (
+    <Alert severity='warning' className={classes.alert}>
+      <Heading level={1} size='xsmall' spacing>
+        {t('process_editor.too_old_version_title')}
+      </Heading>
+      <Paragraph className={classes.alertText} size='small'>
+        <Trans i18nKey={t('process_editor.too_old_version_text')}>
+          Applikasjonen din har versjon <strong>{appLibVersion}</strong> av app-lib-pakken.
+          <br />
+          Versjonen er for gammel for å kunne redigere denne modellen. <br />
+          <br />
+          Du kan se på prosessen, men vi anbefaler at du oppgraderer til versjon 8 eller nyere.
+          <br />
+          <br />
+          Trenger du hjelp til å oppgradere, kan du kontakte{' '}
+          <Link href='servicedesk@altinn.no' target='_new' rel='noopener noreferrer'>
+            servicedesken
+          </Link>{' '}
+          vår.
```

Comment:
What do you think about the suggestion below? This ensures that we got consistent spacing between the paragraph that is defined within the design system. We also abstract every hard-coded text away from the component by using the "t" option param. 😊


```suggestion
   <Heading level={1} size='xsmall' spacing>
        {t('process_editor.too_old_version_title')}
    </Heading>
 <Paragraph spacing>{t('process_editor.too_old_version_text', {  appLibVersion })}</Paragraph>
 <Paragraph spacing>{t('process_editor.need_to_co|ntact_text')}</Paragraph>
 <Link href='servicedesk@altinn.no' target='_new' rel='noopener noreferrer'>{t('general.servicedesk')}</Link>
```

Reply:
That's not possible to have the same design as UX has designed. We need the ```Trans``` to be present because we have a link in the middle of the text. 
However, as Nina suggested in another comment, this might all change now as there are new suggestions for text to be displayed 😄